src/libraries/Common/src/System/Net/Security/Unix/SafeDeleteNegoContext.cs

```diff
@@ -63,7 +65,6 @@ public SafeDeleteNegoContext(SafeFreeNegoCredentials credential, string targetNa
 
         public void SetGssContext(SafeGssContextHandle context)
         {
-            Debug.Assert(context != null && !context.IsInvalid, "Invalid context passed to SafeDeleteNegoContext");
```

Comment:
We now explicitly allow invalid handles to reset the previous values so the `Assert` became useless. `context == null` is already checked by the nullability in the compiler.